src/frontend/magnify/packages/components/src/components/rooms/RegisterRoomForm/RegisterRoomForm.tsx

```diff
@@ -57,6 +59,9 @@ export const RegisterRoomForm = ({ onSuccess }: RegisterRoomFormProps) => {
       });
       onSuccess(newRoom);
     },
+    onError: (error) => {
+      modals.handleNotLoggedRequest(error);
```

Comment:
Why did you named it `handleNotLoggedRequest` and not just `handleErrors` ? How are you sure that it's because the user is not logged and not anything else ?